mbed_project/_internal/project_data.py

```diff
@@ -87,16 +117,19 @@ def from_existing(cls, root_path: Path) -> "MbedProgramData":
             ValueError: no MbedProgramData exists at this path.
         """
         config: Optional[Path]
-        config = root_path / "mbed_app.json"
+        config = root_path / APP_CONFIG_FILE_NAME
         if not config.exists():
             logger.info("This program does not contain an mbed_app.json config file.")
             config = None
 
-        mbed_file = root_path / ".mbed"
-        if not mbed_file.exists():
-            raise ValueError(f"An Mbed program does not exist at {root_path}")
+        mbed_os_file = root_path / MBED_OS_REFERENCE_FILE_NAME
+        if not mbed_os_file.exists():
+            raise ValueError("This path does not contain an mbed-os.lib, which is required for mbed programs.")
```

Comment:
Is it worth mentioning here what the mbed-os.lib file is for? I imagine that if someone has hit this error then they probably don't know.

Reply:
Yes, makes sense. I'll add some more words.

Reply:
I take it back - its explained later on.

Reply:
It's up to you. It looks like the error will be incorporated into the longer explanation you provide in `MbedProgram.from_remote_url()`, but I guess being as explicit as possible can't hurt.

Reply:
Oh yeah, so it is. It's been a long, long weekend and I had forgotten what my own error messages were.